packages/runner-ct/src/app/app.tsx

```diff
@@ -40,13 +45,44 @@ const App: React.FC<AppProps> = observer(
     const [pluginsHeight, setPluginsHeight] = React.useState(500)
     const [isResizing, setIsResizing] = React.useState(false)
     const [isSpecsListOpen, setIsSpecsListOpen] = React.useState(true)
+    const [leftSideOfSplitPaneWidth, setLeftSideOfSplitPaneWidth] = React.useState(DEFAULT_LEFT_SIDE_OF_SPLITPANE_WIDTH)
+    const headerRef = React.useRef()
+
+    function monitorWindowResize () {
+      const $header = $(findDOMNode(headerRef.current))
+
+      function onWindowResize () {
+        state.updateWindowDimensions({
+          windowWidth: window.innerWidth,
+          windowHeight: window.innerHeight,
+          reporterWidth: leftSideOfSplitPaneWidth + VIEWPORT_SIDE_MARGIN,
+          headerHeight: $header.outerHeight(),
+        })
+      }
+
+      $(window)
```

Comment:
Btw I wrote the hook version of this. If you find it helpful it can be easier to use or move it to custom hook

Reply:
I find hooks' magic often difficult to grok.
If this is all the same to you, I would prefer to leave it as it is and refactor later.